pyk4a/pyk4a.py

```diff
@@ -103,34 +122,57 @@ def get_capture(self, timeout=TIMEOUT_WAIT_INFINITE,) -> "PyK4ACapture":
                 in the current capture. There are no guarantees that the returned
                 object will contain all the requested images.
 
-        If using any ColorFormat other than ColorFormat.BGRA32, the color image must be
+        If using any ImageFormat other than ImageFormat.COLOR_BGRA32, the color color_image must be
         decoded. See example/color_formats.py
         """
-        res, capture_capsule = k4a_module.device_get_capture(self._device_id, self.thread_safe, timeout)
-        self._verify_error(res)
-
-        capture = PyK4ACapture(device=self, capture_capsule=capture_capsule)
+        self._validate_is_opened()
+        res, capture_capsule = k4a_module.device_get_capture(self._device_handle, self.thread_safe, timeout)
+        _verify_error(res)
+
+        capture = PyK4ACapture(
+            calibration=self.calibration,
+            capture_handle=capture_capsule,
+            color_format=self._config.color_format,
+            thread_safe=self.thread_safe,
+        )
         return capture
 
     def get_imu_sample(self, timeout: int = TIMEOUT_WAIT_INFINITE) -> Optional["ImuSample"]:
-        res, imu_sample = k4a_module.device_get_imu_sample(self._device_id, self.thread_safe, timeout)
-        self._verify_error(res)
+        self._validate_is_opened()
+        res, imu_sample = k4a_module.device_get_imu_sample(self._device_handle, self.thread_safe, timeout)
+        _verify_error(res)
         return imu_sample
 
+    @property
+    def calibration_raw(self) -> str:
+        self._validate_is_opened()
+        raw = k4a_module.device_get_raw_calibration(self._device_handle, self.thread_safe)
+        return raw
+
+    @calibration_raw.setter
+    def calibration_raw(self, value: str):
+        self._validate_is_opened()
+        self._calibration = Calibration.from_raw(
+            value, self._config.depth_mode, self._config.color_resolution, self.thread_safe
+        )
```

Comment:
Not sure but maybe this should be with class `Calibration`
Maybe allow setting a `Calibration` object? For a future PR possibly.

Reply:
Sure. Let's leave for future.